flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/connector/kafka/sink/KafkaSinkITCase.java

```diff
@@ -143,12 +151,13 @@ public static void setupAdmin() {
         admin = AdminClient.create(properties);
     }
 
-    @AfterClass
+    @AfterAll
     public static void teardownAdmin() {
```

Comment:
```suggestion
    static void teardownAdmin() {
```
could be package private